test/headless/specs/watcher/exposed-event.spec.js

```diff
@@ -40,13 +40,13 @@ testModule('Watcher Exposed Event', class extends WatcherTestClass {
   ['@test should fire twice if moved in, out, and then back in viewport']() {
     return this.context.scrollTo(100)
       .scrollTo(140)
-      .wait(RAF_THRESHOLD)
+      .wait(RAF_THRESHOLD * 2)
```

Comment:
I think it's more clear to keep `RAF_THRESHOLD=16` and then do `RAF_THRESHOLD * 4` here, since `4=skipCounter`.